clients/client-s3/src/schemas/schemas_0.ts

```diff
@@ -3943,7 +3944,7 @@ export var PutObjectRetentionRequest$: StaticStructureSchema = [
   [
     [0, 1],
     [0, 1],
-    [() => ObjectLockRetention$, { [_xN]: _Ret, [_hP]: 1 }],
+    [() => ObjectLockRetention$, { [_hP]: 1, [_xN]: _Ret }],
```

Comment:
todo for later: need to sort the traits before generating to avoid thrashing like this